### ISSUE TITLE
Any plans to continue work on this provider?

### ISSUE THREAD

**Issue by ivica-k** [houqp/terraform-provider-airflow#3]
As the title says, do you have any plans to continue working on this provider? I'm really sorry that I don't know Go and can't help.

**Comment by houqp** [comment]
Yes, this is used in production at work, so i am definitely maintaining it :) Connection support should be fairly straight forward to add, but before you try that path, I recommend giving the new secrets backend a try first if you can fit all your connections into a single AWS secrets manager instance.
